guides/auth-torii-with-github.md

```diff
@@ -466,52 +520,139 @@ module.exports = function(app) {
 ```
 
 It creates a payload out of the `authorizationCode` and optionally `state` sent to it as well as the `client_id` and
-`client_secret` taken from your `.env`. It sends them to the `/login/oauth/access_token` end point,
+`client_secret` taken from your `.env`. It sends them to the `/login/oauth/access_token` endpoint,
 takes GitHub's response and returns it verbatim. The resulting access token will be in an `access_token` property of
 the JSON. We aren't going to run this through Ember Data, so the result does not need to be JSONAPI.
 
+#### Using Gatekeeper
+
+This aspect of the guide takes place outside of the context of your Ember app,
+because GitHub will not permit token exchange to happen on a client-side
+application. Luckily, spinning up a separate gatekeeper server is somewhat
+straightforward, and there even is a Node application,
+[Gatekeeper](prose/gatekeeper), designed to make the process even easier.
+
+In short, the server must receive the authorization code your Ember app receives
+from GitHub and forward it back to GitHub in a `POST` request, wait for a
+response, and send it back to your Ember app.
+
+Gatekeeper expects the authorization code to come at the end of a URL in the
+format of `http://<gatekeeper-server>/authenticate/<authorization-code>`,
+which is why we set the value of `DEV_TOKEN_EXCHANGE_URL` to
```

Comment:
Should this one also be `AUTH_TOKEN_EXCHANGE_URL`?

Reply:
yes 👍